### ISSUE TITLE
Add support for CSS selector level 4 :not()

### ISSUE THREAD

**Issue by MoOx** [cssnext/cssnext#106]
In css3, not handle on selector only. Now it can take multiple values

> The :not() pseudo-class first appeared in CSS3, when it only allowed a maximum of 1 simple selector as an argument. In CSS4, it can take a list of selectors as an argument. The styles are applied to the element which is not represented by the passed argument.

http://css4.rocks/css4-selectors/negation-pseudo-class.php

So I guess we can transform this

```css
p:not(:first-child, .special)
{}
```

into

```css
p:not(:first-child),
p:not(.special)
{}
```

**Comment by kud** [comment]
:+1:

**Comment by magsout** [comment]
:+1:

**Comment by MoOx** [comment]
BAM https://github.com/postcss/postcss-selector-not

**Comment by MoOx** [comment]
Closed by 64b903f6742ab7eed826fd1400161fbde0e2f9a9

**Comment by MoOx** [comment]
Released as 1.4.0
